src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_up_commands.py

```diff
@@ -905,5 +905,100 @@ def test_webapp_up_generate_default_name(self, resource_group):
         shutil.rmtree(temp_dir)
 
 
+    @live_only()
```

Comment:
I'd like to ask why this test is ` @live_only()`?

Reply:
The test case was modeled after a pre-existing test function which tested the ability to use `az webapp up` to deploy a python webapp (linux). I modified it to also deploy a windows app to the same resource group.  The original test function, as well as most of the other `az webapp up` tests, are live only as well -- perhaps that's not a solid rationale though

@calvinsID @panchagnula do you know why most of these tests are live only and have any thoughts on whether they should be or not?

Reply:
@zhoxing-ms the az webapp up is a not idempotent like az webapp create i.e a simple flow is
1. webapp doesn't exist, we create the webapp, ASP, RG & deploy the code to the webapp
2. App exists, the code path is to deploy contents
 az webapp up was done as a quick start command, for users who are completely new to azure & also to have a command with minimal parameters. Hope this helps.

Reply:
Got it, thanks~